protocols/ping/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.43.2
+- Calling `no_keep_alive` on every Stream received.
+  See [PR 4595](https://github.com/libp2p/rust-libp2p/pull/4595).
```

Comment:
This is only an internal change so we don't need to document it in the changelog.

```suggestion
## 0.43.2 - unreleased

<!-- Only internal changes so far, please delete this comment with the next user-facing change. -->

```

Reply:
remove internal change from changelog 412aa75